kratos/tests/auxiliar_files_for_python_unittest/mdpa_files/assign_mpcs_to_neighbours_process.mdpa

```diff
@@ -0,0 +1,3308 @@
+Begin Properties 0
```

Comment:
Please, can you consider a smaller mesh?, you can put this one in the Example repository